.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -21,13 +21,20 @@ If applicable, add screenshots to help explain your problem.
 
 ### Operating System
 
-Please consider your privacy, before completing this section.  
+*Please consider your privacy, before completing this section.*
+
 Example: Ubuntu 16.04
 
 ### Logs
 
-Please consider your privacy, before completing this section.  
-You can find the `Logs` file in the dropdown menu of the search bar or inside your Wasabi data folder.
+*Please consider your privacy, before completing this section.*
+
+```
+Insert Your Logs Here
+```
+
+
+You can find the `Logs.txt` file in the dropdown menu of the search bar or inside your Wasabi data folder or find `%PROFILE_PATH%/WalletWasabi/Client/Logs.txt` file.
```

Comment:
Here I would extend the path with:
`%PROFILE_PATH%/AppData/Roaming/WalletWasabi/Client/Logs.txt`.

Reply:
True, true, mine is windows specific, not everyone uses that. Leave it as it is.